RateExchange/src/API/FetchAPI.js

```diff
@@ -0,0 +1,75 @@
+import * as LocalStorage from "../Utils/LocalStorage"; 
+
+export const fetchAPILatest = (currency) => {
+    
+    return (
+        fetch(`https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest/currencies/${currency}.json`)
```

Comment:
Do you really need two different functions?
Can you combine fetchAPILAtest with fetchAPIDate?